examples/sveltekit/playwright.config.js

```diff
@@ -0,0 +1,10 @@
+/** @type {import('@playwright/test').PlaywrightTestConfig} */
+const config = {
```

Comment:
Do all templates start with playwright now?

Reply:
We could generate it without, but it's the "blessed integration testing framework" from Kit's perspective.